demisto_sdk/commands/content_graph/interface/neo4j/neo4j_graph.py

```diff
@@ -539,17 +538,9 @@ def import_graph(self, imported_path: Optional[Path] = None) -> bool:
                 session.execute_write(merge_duplicate_content_items)
                 session.execute_write(create_constraints)
                 session.execute_write(remove_empty_properties)
-        try:
-            # Test that the imported graph is valid by marshaling it
-            self.marshal_graph(MarketplaceVersions.XSOAR)
-            result = True
-        except ValidationError as e:
-            logger.warning("Failed to import the content graph")
-            logger.debug(f"Validation Error: {e}")
-            result = False
-        # clear cache after loading the graph
+        has_infra_graph_been_changed = self._has_infra_graph_been_changed()
         self._id_to_obj = {}
-        return result
```

Comment:
is it possible to :
self._id_to_obj = {}
return self._has_infra_graph_been_changed()

Reply:
I agree with your point.
but I didn't want to change it because I'm afraid of changing unnecessary things here.